sklearn/utils/sparsetools/setup.py

```diff
@@ -1,21 +1,18 @@
 import numpy
 
 
+
```

Comment:
Interestingly flake8_diff.sh is not able to spot these. Quickly looking at it this is because we do the diff without context (`--unified=0`) in order to avoid noise coming from lines that the PR hasn't touched.